install.sh

```diff
@@ -46,14 +47,33 @@ echo ""
 printf "Installing docker images for ddev to use...\n"
 
 if command -v xzcat >/dev/null; then
-    xzcat ddev_tarballs/ddev_docker_images*.tar.xz | docker load
+    ${XZCAT} ddev_tarballs/ddev_docker_images*.tar.xz | docker load
 elif [[ "$OS" == "Darwin" ]]; then
-    gzip -dc ddev_tarballs/ddev_docker_images*.tar.xz | docker load
+    XZCAT="gzip -dc"
+    ${XZCAT} ddev_tarballs/ddev_docker_images*.tar.xz | docker load
 else
     printf "${YELLOW}Unable to load ddev_docker_images. They will load at first 'ddev start'.${RESET}\n"
 fi
 
 
+# Allows the user to choose whether they need an editor with PHP-capabilities.
+if [ -f ddev_tarballs/docker_additions.tar.xz ]; then
```

Comment:
I need to figure out how to do something similar in package.sh here in install.sh.